### PR TITLE
small_fixes_bugs

### PR DESCRIPTION
**Pull request by tantting**
## Description

Fixed bugs with exchangerates, loan and the transaction log but also smaller fixes. 

## Related Issue ( optional )

Link to any related issue

## Changes

Describe the changes in more detail, if necessary include explanation of why some choices were made.

## Screenshots & GIFs ( optional )

media to provide more visual context if necessary

## Notes & Comments

General notes and comments, such as if there is things to consider ahead of deployment

### CODE REVIEW

File: RebelAllianceBank/Accounts/CardAccount.cs [hunk 1]
```diff
@@ -35,36 +35,62 @@ public void AddToTransactionLog(Transaction newTransaction)
...
         public void ShowTransactionLog()
         {
+             _transactionsLog.Reverse();
+            Console.WriteLine("-------------------------------------------------------------------------------------");
+            Console.WriteLine($"Nuvarande saldo på konto: {this.Balance:N2} {AccountCurrency}");
+            Console.WriteLine("-------------------------------------------------------------------------------------");
             _transactionsLog.Reverse();
-            Console.WriteLine($"Nuvarande saldo på konto: {this.Balance}");
-            Console.WriteLine("---------------------------------------------------");
+            const string format = "{0,-30} {1,-40} {2, -30}";
```
Comment (sidstedt) [inline RebelAllianceBank/Accounts/CardAccount.cs:43]: nicely done!

File: RebelAllianceBank/Accounts/CardAccount.cs [hunk 2]
```diff
@@ -35,36 +35,62 @@ public void AddToTransactionLog(Transaction newTransaction)
 
         public void ShowTransactionLog()
         {
+             _transactionsLog.Reverse();
+            Console.WriteLine("-------------------------------------------------------------------------------------");
+            Console.WriteLine($"Nuvarande saldo på konto: {this.Balance:N2} {AccountCurrency}");
+            Console.WriteLine("-------------------------------------------------------------------------------------");
             _transactionsLog.Reverse();
```
Comment (sidstedt) [inline RebelAllianceBank/Accounts/CardAccount.cs:42]: remove one _transactionsLog.Reverse();, from each of the three classes. As you call it two times before the foreach loop and one time after. And it should only be one before and after, right?

**Review by sidstedt — COMMENTED** [review]
check comment :)

**Review by sidstedt — APPROVED** [review]
Good job!

File: RebelAllianceBank/Accounts/CardAccount.cs (hunk 2 above)
Comment (tantting) [reply RebelAllianceBank/Accounts/CardAccount.cs]: Great find! Fixed

**Pull request merged**